src/main/java/com/elastic/support/monitoring/MonitoringExportInputs.java

```diff
@@ -132,8 +132,7 @@ public boolean runInteractive() {
             return Collections.singletonList("Invalid Date or Time format. Please enter the date in format YYYY-MM-dd HH:mm");
         }
 
-        return emptyList;
-
+        return null;
```

Comment:
Careful with returning `null` instead of an empty list. Some parts of the code expect things that they cannot safely expect.

Given that this code relates explicitly to fetching stack monitoring data, I am unsure if you tested this behavior.

Reply:
Done.

This PR also fixes a nasty bug. When `port` is invalid, many port validation errors are printed out. This was happening because `emptyList` is mutable and shared. I have another PR in the queue to avoid those issues in the future.